cloud-watch-to-slack-testing/deployment/index.js

```diff
@@ -246,6 +246,29 @@ function s3ActivityMessageText(message) {
   return `${userName} generated S3 event ${eventName} from region ${awsRegion} for bucket ${bucketName} in Dockstore ${dockstoreEnvironment}`;
 }
 
+function ecsActivityMessageText(message) {
+  if (message["detail-type"] === "ECS Task State Change") {
+    return ecsTaskStateChangeMessageText(message);
+  } else {
+    return ecsAutoScalingMessageText(message);
+  }
+}
+
+function ecsTaskStateChangeMessageText(message) {
+  const taskArn = message?.detail?.taskArn;
+  const lastStatus = message?.detail?.lastStatus;
+  let messageText = `Task ${taskArn} is now ${lastStatus}`;
+  if (message?.detail != undefined) {
+    ["startedAt", "stoppedAt", "stoppedReason"].forEach((name) => {
+      const value = message?.detail[name];
```

Comment:
As I browsed regarding optional chaining, I noticed that there's a syntax `a?.[expr]` that "optional chains" the expression-based lookup.  I am going to make a quick mod to eliminate the outer `if`

Reply:
(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)